src/registrar/services/dns_host_service.py

```diff
@@ -207,11 +221,38 @@ def save_db_zone(self, vendor_zone_data, domain_name):
             dns_domain = Domain.objects.get(name=domain_name)
 
             dns_zone, _ = DnsZone.objects.get_or_create(dns_account=dns_account, domain=dns_domain, name=zone_name)
```

Comment:
Can this also be a straight up `.create` since this runs in a transaction, and it wouldn't already exist (for the `get` part)?